bin/sngl/run_daily_pycbc_export_censored_trigs

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
-# Copyright (C) 2016 Christopher M. Biwer
```

Comment:
Uhh... If you want to add yourself as contributing to some code feel free to add your name to the list. But do not delete people's names.